pkg/activator/net/revision_backends_test.go

```diff
@@ -54,7 +54,7 @@ import (
 const (
 	testNamespace      = "test-namespace"
 	testRevision       = "test-revision"
-	informerRestPeriod = 2 * time.Second
+	informerRestPeriod = 500 * time.Millisecond
```

Comment:
Intentional?

Reply:
Yes. I am not brave enough to remove it completely.